conformance-suites/2.0.0/conformance/extensions/webgl-compressed-texture-etc.html

```diff
@@ -138,10 +139,18 @@
     shouldBe("formats.length", isPositive ? "10" : "0");
 
     debug("");
-    shouldThrow("gl.compressedTexImage2D(gl.TEXTURE_2D, 0, COMPRESSED_R11_EAC, 4, 4, 0, null)");
-    shouldThrow("gl.compressedTexSubImage2D(gl.TEXTURE_2D, 0, 0, 0, 0, 0, COMPRESSED_R11_EAC, null)");
-    shouldThrow("gl.compressedTexImage3D(gl.TEXTURE_2D_ARRAY, 0, COMPRESSED_R11_EAC, 4, 4, 4, 0, null)");
-    shouldThrow("gl.compressedTexSubImage3D(gl.TEXTURE_2D_ARRAY, 0, 0, 0, 0, 0, 0, 0, COMPRESSED_R11_EAC, null)");
+    if (contextVersion >= 2) {
+      // `null` coerces into `0` for the PBO entrypoint, yielding INVALID_OP due to no PBO bound.
```

Comment:
@zhenyao , If we want to list the definition of compressedTexImage2D:
 1) void compressedTexImage2D(GLenum target, GLint level, GLenum internalformat, GLsizei width,
                            GLsizei height, GLint border, GLsizei imageSize, GLintptr offset);
  2) void compressedTexImage2D(GLenum target, GLint level, GLenum internalformat, GLsizei width,
                            GLsizei height, GLint border, ArrayBufferView srcData,
                            optional GLuint srcOffset = 0, optional GLuint srcLengthOverride = 0);

If we only have 7 parameters and call compressedTexImage2D, the 7th parameters should be always "ArrayBufferView", which is required, not optional. So if it can not be "null".  However, in texImage2D, the same parameter is  "ArrayBufferView?", it can be null in this case. 

I suppose that both you and Jeff want to use the first definition of compressedTexImage2D, so we need 8 parameters. I have updated the fix at https://github.com/KhronosGroup/WebGL/pull/2253, could you take another look? 

@kainino0x , no problem. :)